src/betterproto/__init__.py

```diff
@@ -1191,16 +1191,15 @@ def from_dict(self: T, value: Dict[str, Any]) -> T:
                             ]
                         else:
                             v = [cls().from_dict(item) for item in value[key]]
-                    elif isinstance(v, datetime):
+                    elif cls == datetime:
```

Comment:
Can you change these to if cls is datetime?

Reply:
For whatever reason, there are a lot of instances of `cls == foo` in this file and none of `cls is foo`. I think it makes sense to maintain consistency but I'm happy to change all of them to use `is` if you prefer.

Reply:
Done

Reply:
I agree with both of you, and it was courteous of you to match the existing style, @emosenkis . I'd say given that it's a style point, leave as-is and fix on a refactor.